ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2

```diff
@@ -17,6 +20,9 @@ ExecStop=/usr/bin/docker stop syncd
 ExecStopPost=/etc/mlnx/msn2700 stop
 ExecStopPost=/etc/init.d/sxdkernel stop
 ExecStopPost=/usr/bin/mst stop
+{% elif sonic_hwsku == 'AS7512' %}
+ExecStopPost=/etc/init.d/xpnet.sh stop
+ExecStopPost=/etc/init.d/xpnet.sh start
```

Comment:
@stcheng for example for ptf testing (when syncd stopped and saiserver deployed)